is-wds-admin.php

```diff
@@ -84,20 +87,40 @@ public static function get_instance() {
 	}
 
 	/**
-	 * Sets up our plugin
+	 * Sets up our plugin.
 	 *
 	 * @since  1.0.0
 	 */
 	protected function __construct() {
 		$this->basename        = plugin_basename( __FILE__ );
+		$this->privileged_user = 'wds_admin'; // This can be whatever user is the privileged user.
+		$this->version         = $this->get_version();
+
+		// Add capability if it does not exist for the given user.
+		add_action( 'init', array( $this, 'add_cap_if_not_exists' ) );
+	}
+
+	/**
+	 * Get the version of this plugin.
+	 *
+	 * @author Aubrey Portwood
+	 * @since 1.0.0
+	 *
+	 * @return string The Version value in the header.
+	 */
+	private function get_version() {
+		$headers = (object) get_file_data( __FILE__, array(
```

Comment:
It would only happen if this method was called again, and it's called only once when the class is created, and it's only created once.